From ee2d9a6b410ecebecd021df454156e96951ca316 Mon Sep 17 00:00:00 2001
From: hienhuynh <hien.huynh.px@rvc.renenas.com>
Date: Mon, 9 Apr 2018 10:44:41 +0700
Subject: [PATCH 481/628] ARM: DTS: r8a7742: add reg-names for sound

Rensas R-Car sound driver supports reg-names.
Let's use it.

Signed-off-by: hienhuynh <hien.huynh.px@rvc.renenas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 arch/arm/boot/dts/r8a7742.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7742.dtsi b/arch/arm/boot/dts/r8a7742.dtsi
index 04d019a..336ce2b 100644
--- a/arch/arm/boot/dts/r8a7742.dtsi
+++ b/arch/arm/boot/dts/r8a7742.dtsi
@@ -1387,6 +1387,9 @@
 			<0 0xec5a0000 0 0x100>,  /* ADG */
 			<0 0xec540000 0 0x1000>, /* SSIU */
 			<0 0xec541000 0 0x1280>;  /* SSI */
+
+		reg-names = "scu", "adg", "ssiu", "ssi";
+
 		clocks =	<&mstp10_clks R8A7742_CLK_SSI_ALL>,
 				<&mstp10_clks R8A7742_CLK_SSI9>,
 				<&mstp10_clks R8A7742_CLK_SSI8>,
-- 
2.7.4

